### DISCUSSION TITLE
Styles are not loaded when using standalone

### DISCUSSION THREAD

**Discussion by kamolovd** [vercel/next.js#68064] [App Router]
I use standalone for the nextjs project to load the standalone folder into the server. But the styles disappear after the build and launch of server.js in the .next/standalone path. After I added this config and now when building I get an error
<img width="535" alt="image" src="https://github.com/user-attachments/assets/7502071a-b970-4e9e-94bd-49dab3b635aa">

Error
Module parse failed: Unexpected token (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> :root {
|   --max-width: 1100px;
|   --border-radius: 12px;

Import trace for requested module:
./app/styles/globals.css
./app/layout.tsx

**Comment by Juneezee** [comment]
You shouldn't need to configure Webpack. Next.js already handles this automatically.

With `output: 'standalone'`, when you run `next build`, Next.js will create `.next/standalone` and `.next/static` folders. You need to copy the `.next/static` folder into `.next/standalone/.next` folder. See [Automatically Copying Traced Files](https://nextjs.org/docs/app/api-reference/next-config-js/output#automatically-copying-traced-files).

```
$ next build
$ mv .next/static .next/standalone/.next
$ node .next/standalone/server.js

  ▲ Next.js 14.2.5
  - Local:        http://localhost:3000
  - Network:      http://0.0.0.0:3000

 ✓ Starting...
 ✓ Ready in 118ms
```

Your `.next/standalone` folder should look like this:

![image](https://github.com/user-attachments/assets/58edca3f-9af6-43b8-a50a-fb13995fa2df)
